### PR TITLE
da: celestia-openrpc - use blob commitment

### PR DESCRIPTION
**Pull request by tuxcanfly**
## Overview

This PR updates the `FrameRef` to use a blob commitment using the `BlobModule` API from `celestia-openrpc`.

While here also added a unit test for `FrameRef` serialization / deserialization.

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by tuxcanfly** [comment]
We can use `BlobModule` API `Submit(blobs []*blob.Blob)uint64` so that we don't have to hardcode the gas and fee.

**Comment by jcstein** [comment]
are there any changes to existing UX? [here](https://docs.celestia.org/developers/optimism-devnet/) or [here](https://docs.celestia.org/developers/optimism/). i don't think so

File: op-celestia/celestia/celestia_test.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package celestia
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestEncodeDecodeFrameRef(t *testing.T) {
```
Comment (MSevey) [inline op-celestia/celestia/celestia_test.go:10]: For these types of table tests, it is good to test valid and invalid test cases. 

So I would add a `isValid` field or something similar, so then in the test below, you can verify invalid cases do return an error.

**Review by MSevey — CHANGES_REQUESTED** [review]
utACK, just add an invalid test case to the test so we can be even more confident in the encoding and decoding.

**Comment by tuxcanfly** [comment]
> are there any changes to existing UX? [here](https://docs.celestia.org/developers/optimism-devnet/) or [here](https://docs.celestia.org/developers/optimism/). i don't think so

Nope. We did remove hardcoded gas and fee but that wasn't documented before anyway.

File: op-celestia/celestia/celestia_test.go (hunk 1 above)
Comment (tuxcanfly) [reply op-celestia/celestia/celestia_test.go:10]: Added 👍🏽

**Comment by tuxcanfly** [comment]
> utACK, just add an invalid test case to the test so we can be even more confident in the encoding and decoding.

Done 👍🏽

**Review by MSevey — APPROVED** [review]

**Pull request merged**